server/app.js

```diff
@@ -24,10 +27,13 @@ if (app.get("env") === "production") {
 	app.use(httpsOnly());
 }
 
-app.use(apiRoot, router);
+// using middleware in order to routing
+app.use("/api/user", userRoute);
+app.use("/api/stars", starRoute)
+app.use("/api/auth", authRoute)
```

Comment:
+1 this is good clean structuring of your APIs based on responsibility. Good work!